iceoryx_examples/iceperf/uds.cpp

```diff
@@ -23,16 +23,55 @@
 #include <thread>
 
 UDS::UDS(const std::string& publisherName, const std::string& subscriberName) noexcept
-    : m_publisherName(publisherName)
-    , m_subscriberName(subscriberName)
+    : m_publisherSocketName(PREFIX + publisherName)
```

Comment:
Fascinating. In structs you use the curly braces for initialization and in ctors you don't. It is like a programmers fingerprint. I prefer the assignment (=) in structs and curly braces in ctors.

Reply:
Well, there are some places where `=` doesn't work but `{}` does. In the initialization list `()` also always works, so I never changed my habit. But good point, we should probably encourage to use `{}` everywhere ... except when macros are involved, like `TIMING_TEST` which gets confused by the curly braces.